tests/spec/core/markdown-spec.js

```diff
@@ -1,5 +1,5 @@
 "use strict";
-describe("Core - Markdown", function() {
+fdescribe("Core - Markdown", function() {
```

Comment:
no fdescribe!

Reply:
nah, it worked (locally)... but I didn't hit save in my editor it seems 😅